lib/src/main/proto/bio/terra/tanagra/underlay/tables.proto

```diff
@@ -48,3 +48,18 @@ message ColumnId {
   // The name of the column.
   string column = 3;
 }
+
+// A filter specification to get a subset of the rows in a table.
+message TableFilter {
+  // A condition on a particular column
+  // TODO: allow multiple ColumnFilters and AND/OR'ing them together
+  ColumnFilter column_filter = 1;
+}
+
+// A building block for a TableFilter that defines a condition on a single column.
+message ColumnFilter {
+  // Pointer to the column.
+  ColumnId column = 1;
```

Comment:
Here's where we define a `ColumnId` property in the proto definition for `ColumnFilter`.

Reply:
Defining these as Column IDs and then resolving into a richer Column representation in the Autovalue table seems reasonable to me. Looks consistent with what is being done elsewhere in the underlay conversion, e.g. hierarchical references. Am I missing something?

Reply:
Yeah I was confused why there are both `Column` and `ColumnId` Autovalue classes. But I think it's just easier if the serialization classes generally line up. e.g. We could have a `Column` class that has an `id` property, which we use wherever the `ColumnId` is in the underlay proto, instead of deserializing into two separate objects. I can see how that could be harder to follow the deserialization flow from underlay -> internal classes though.